userService/stripe_email_updater.go

```diff
@@ -0,0 +1,87 @@
+package userService
+
+import (
+	"bytes"
+	"eth2-exporter/db"
+	"eth2-exporter/utils"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/lib/pq"
+)
+
+func stripeEmailUpdater() {
+	for {
+		// fetch all users with pending stripe email updates
+		var pendingUsers []struct {
+			Email            string `db:"email"`
+			StripeCustomerId string `db:"stripe_customer_id"`
+		}
+		err := db.FrontendReaderDB.Select(&pendingUsers, "SELECT email, COALESCE(stripe_customer_id. '') FROM users WHERE stripe_email_pending")
```

Comment:
Super minor, I understand your point about throwing an error and this query is probably fine for our users, so feel free to ignore. But limiting the select on stripe_customer_id would be more performant due to the unique constraint, no full table scan needed. But we can always change this later on if we are unhappy with the performance at some point